<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveProfileManagerWorkPanel</class>
 <widget class="QWidget" name="ActiveProfileManagerWorkPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Active Profile</string>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QWidget" name="projectNameW" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Project:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="projectPB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0px none; 
	background-color: #FFFF7F;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Active profile:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QPushButton" name="activeProfilePB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 0px none; 
	background-color: #FFFF7F;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true"> QPushButton {
     border: 1px solid #dadbde;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
     border-radius: 2px;
     min-width: 34px;
     min-height: 24px;
 }

 QPushButton:pressed {
     border: 2px solid #777777;
 }

 QPushButton:checked {
     border: 2px solid #777777;
 }</string>
     </property>
     <property name="title">
      <string>Profile objects</string>
     </property>
     <layout class="QVBoxLayout" name="_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="profileTV">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="controlW3" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="profileNewObjectPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>New object</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-new.svg</normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-new.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="removeProfileObjectPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Remove object</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-remove.svg</normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-remove.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="profileObjectShowHidePB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Show/Hide object</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-showhide.svg</normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-showhide.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="profileObjectUpPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Move object up</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/go-up.svg</normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/go-up.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="profileObjectDownPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Move object down</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/go-down.svg</normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/go-down.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="deleteProfileObjectPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Delete object</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-delete.svg</normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-delete.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="profileObjectInfoPB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Object info</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-info.svg</normaloff>:/images/StandardWP/GraphicsManagerWorkPanel/object-info.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
